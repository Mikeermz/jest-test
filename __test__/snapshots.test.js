import { getCharacter } from '../snapshots';
import rick from '../rick.json';

describe('Realizando Instantaneas', () => {
  test('Snapshot', () => {
    expect(getCharacter(rick)).toMatchSnapshot();
  });

  test('Siempre va a fallar', () => {
    const user = {
      name: "Mikee Ramirez",
      id: Math.floor(Math.random() * 20)
    }
    expect(user).toMatchSnapshot();
  });
  test('Aceptando excepciones', () => {
    const user = {
      name: "Mikee Ramirez",
      id: Math.floor(Math.random() * 20)
    }
    console.log(user)
    expect(user).toMatchSnapshot({
      id: expect.any(Number)
    });
  });
});