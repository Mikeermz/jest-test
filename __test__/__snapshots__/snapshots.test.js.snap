// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Realizando Instantaneas Aceptando excepciones 1`] = `
Object {
  "id": Any<Number>,
  "name": "Mikee Ramirez",
}
`;

exports[`Realizando Instantaneas Siempre va a fallar 1`] = `
Object {
  "id": 13,
  "name": "Mikee Ramirez",
}
`;

exports[`Realizando Instantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanches",
  "species": "Human",
  "status": "Alive",
}
`;
